src/app/shared/my-journey-ongoing-card/my-journey-ongoing-card.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="ongoing-card" (click)="navigateproject(data._id)">
+  <div class="ongoing-head">
+    <div>
+      @if (data.endDate) {
```

Comment:
use safety operators wherever you're trying to get data from object like this

Reply:
done